scripts/dbgen/main.go

```diff
@@ -160,6 +165,78 @@ const (
 	return os.WriteFile(outputPath, data, 0o600)
 }
 
+// generateForeignKeyConstraints generates the ForeignKeyConstraint enum.
+func generateForeignKeyConstraints() error {
+	localPath, err := localFilePath()
+	if err != nil {
+		return err
+	}
+	databasePath := filepath.Join(localPath, "..", "..", "..", "coderd", "database")
+
+	dump, err := os.Open(filepath.Join(databasePath, "dump.sql"))
+	if err != nil {
+		return err
+	}
+	defer dump.Close()
+
+	var foreignKeyConstraints []string
+	dumpScanner := bufio.NewScanner(dump)
+	query := ""
+	for dumpScanner.Scan() {
+		line := strings.TrimSpace(dumpScanner.Text())
+		switch {
+		case strings.HasPrefix(line, "--"):
+		case line == "":
+		case strings.HasSuffix(line, ";"):
+			query += line
+			if strings.Contains(query, "FOREIGN KEY") {
+				foreignKeyConstraints = append(foreignKeyConstraints, query)
+			}
+			query = ""
+		default:
+			query += line + " "
+		}
+	}
+
+	if err := dumpScanner.Err(); err != nil {
+		return err
+	}
+
+	s := &bytes.Buffer{}
+
+	_, _ = fmt.Fprint(s, `// Code generated by gen/enum. DO NOT EDIT.
```

Comment:
what is `gen/enum`?

Reply:
Seems copy-paste/outdated from when this was refactored into a single `dbgen` command.

Reply:
I'm not sure, just following the existing convention. I can update to `scripts/dbgen` if it's more clear.

Reply:
Feel free to update this and other places, if anything says other than `scripts/dbgen` (or just `dbgen`) then it's wrong. 😄